src/main/java/org/helpberkeley/memberdata/v200/RestaurantBeanV200.java

```diff
@@ -209,6 +221,11 @@ private String getColumnName(final String fieldName) {
 
     // Utility accessors
 
+    public String[] getFormulas() {
+        return new String[]{getPhone(), getAltPhone(), getNeighborhood(), getCity(), getAddress(),
+                getCondo(), getDetails()};
+    }
+
```

Comment:
Change to return List<String>.  Implementation can use List.of, or construct and populate a new ArrayList<String>